Doc/library/os.rst

```diff
@@ -1230,6 +1231,16 @@ or `the MSDN <https://msdn.microsoft.com/en-us/library/z0kc8e3z.aspx>`_ on Windo
    .. versionadded:: 3.7
 
 
+.. data:: RWF_APPEND
+
+   Append data to the end of the file. See the description of the flag 'O_APPEND' in ''os.open()''.
```

Comment:
'O_APPEND' should be written :data:`O_APPEND`.

''os.open()'' should be written :func:`os.open` to add a link.

But os.open() doesn't document O_APPEND, just "The above constants are available on Unix and Windows"...

I would prefer to copy preadv Linux manual page:

```
       RWF_APPEND (since Linux 4.16)
              Provide a per-write equivalent of  the  O_APPEND  open(2)  flag.
              This  flag  is  meaningful  only  for pwritev2(), and its effect
              applies only to the data range written by the system call.   The
              offset argument does not affect the write operation; the data is
              always appended to the end of the file.  However, if the  offset
              argument is -1, the current file offset is updated.
```

(Adapt it to Python names.)